src/streamingCalls/streamDescriptor.ts

```diff
@@ -41,9 +41,11 @@ import {StreamingApiCaller} from './streamingApiCaller';
  */
 export class StreamDescriptor implements Descriptor {
   type: StreamType;
+  streaming: boolean; // needed for grpc-fallback
```

Comment:
for `grpc-fallback` or for browser support?

If the latter, let's not confuse people with implementation details (like using `grpc-fallback` protocol) of a major environment/feature. If I read this code I'd go and Google/code search "grpc-fallback streaming" expecting to see my answer (probably just get confused). If I searched "browser streaming" instead, I think it would produce better results, especially in code search.

Reply:
"browser support" sounds good to me.